app/models/energy/construction.py

```diff
@@ -338,79 +394,79 @@ class EnergyWindowMaterialBlind(BaseModel):
+    front_beam_solar_reflectance: float=Schema(
         0.5,
         ge=0,
         lt=1,
         description='The beam solar reflectance of the front side of the slat, it is'
         ' assumed to be independent of the angle of incidence. Default value is 0.5.'
     )
 
-    back_beam_solar_reflectance: float = Schema(
+    back_beam_solar_reflectance: float=Schema(
         0.5,
         ge=0,
         lt=1,
         description='The beam solar reflectance of the back side of the slat, it is'
         ' assumed to be independent of the angle of incidence. Default value is 0.5.'
     )
 
-    diffuse_solar_transmittance: float = Schema(
+    diffuse_solar_transmittance: float=Schema(
         0,
         ge=0,
         lt=1,
         description='The slat transmittance for hemisperically diffuse solar radiation.'
         ' Default value is 0.'
     )
 
-    front_diffuse_solar_reflectance: float = Schema(
+    front_diffuse_solar_reflectance: float=Schema(
         0.5,
         ge=0,
         lt=1,
         description='The front-side slat reflectance for hemispherically diffuse solar'
         ' radiation. Default value is 0.5.'
     )
 
-    back_diffuse_solar_reflectance: float = Schema(
+    back_diffuse_solar_reflectance: float=Schema(
         0.5,
         ge=0,
         lt=1,
         description='The back-side slat reflectance for hemispherically diffuse solar'
         ' radiation. Default value is 0.5.'
     )
 
-    beam_visible_transmittance: float = Schema(
+    beam_visible_transmittance: float=Schema(
         0,
         ge=0,
         lt=1,
-        description='The beam visible tranmsittance of the slat, it is assumed to be'
+        description='The beam visible transmittance of the slat, it is assumed to be'
         ' independent of the angle of incidence. Default value is 0.'
     )
 
-    front_beam_visible_reflectance: float = Schema(
+    front_beam_visible_reflectance: float=Schema(
         0.5,
         ge=0,
         lt=1,
         description='The beam visible reflectance on the front side of the slat, it is'
         ' assumed to be independent of the angle of incidence. Default value is 0.5.'
     )
 
-    back_beam_visible_reflectance: float = Schema(
+    back_beam_visible_reflectance: float=Schema(
         0.5,
         ge=0,
         lt=1,
         description='The beam visible reflectance on the back side of the slat, it is'
         ' assumed to be independent of the angle of incidence. Default value is 0.5.'
     )
 
-    diffuse_visible_transmittance: float = Schema(
+    diffuse_visible_transmittance: float=Schema(
         0,
         ge=0,
-        le=1,
+        lt=1,
         description='The slat transmittance for hemispherically diffuse visible'
         ' radiation. This value should equal “Slat Beam Visible Transmittance.”'
     )
 
-    front_diffuse_visible_reflectance: float = Schema(
+    front_diffuse_visible_reflectance: float=Schema(
```

Comment:
We should follow the convention here of not using the word `front` in the name (only for `back`)